code/game/objects/items/miscellaneous.dm

```diff
@@ -101,6 +101,51 @@
 	new /obj/item/toy/crayon/spraycan(src)
 	new /obj/item/clothing/shoes/sandal(src)
 
+/obj/item/choice_beacon/augments
+	name = "augment beacon"
+	desc = "Summons augmentations. Can be used 3 times!"
+	var/uses = 3
+
+/obj/item/choice_beacon/augments/generate_display_names()
+	var/static/list/augment_list
+	if(!augment_list)
+		augment_list = list()
+		var/list/templist = list(
+		/obj/item/organ/cyberimp/brain/anti_drop,
+		/obj/item/organ/cyberimp/arm/toolset,
+		/obj/item/organ/cyberimp/arm/surgery,
+		/obj/item/organ/cyberimp/chest/thrusters,
+		/obj/item/organ/lungs/cybernetic/upgraded,
+		/obj/item/organ/liver/cybernetic/upgraded) //cyberimplants range from a nice bonus to fucking broken bullshit so no subtypesof
+		for(var/V in templist)
+			var/atom/A = V
+			augment_list[initial(A.name)] = A
+	return augment_list
+
+/obj/item/choice_beacon/augments/generate_options(mob/living/M)
+	var/list/display_names = generate_display_names()
+	if(!display_names.len)
+		return
+	var/choice = input(M,"Which item would you like to order?","Select an Item") as null|anything in display_names
+	if(!choice || !M.canUseTopic(src, BE_CLOSE, FALSE, NO_TK))
+		return
+
+	spawn_option(display_names[choice],M)
+	uses--
+	if(!uses)
+		qdel(src)
+	else
+		to_chat(M, "<span class='notice'>[uses] use[uses > 1 ? "s" : ""] remaining on the [src].</span>")
+
+/obj/item/choice_beacon/augments/spawn_option(obj/choice,mob/living/M)
+	new choice(get_turf(M))
+	var/msg = "<span class=danger>Something warps to your feet!</span>"
+	if(ishuman(M))
+		var/mob/living/carbon/human/H = M
+		if(istype(H.ears, /obj/item/radio/headset))
+			msg = "You hear something crackle in your ears for a moment before a voice speaks.  \"Please stand by for a message from S.E.L.F. Message as follows: <span class='bold'>Item request received. Your package has been transported, use the autosurgeon supplied to apply the upgrade.</span> Message ends.\""
```

Comment:
Why is the company that's helping the synths also helping the people who are hunting them?

Also, what autosurgeon? This item doesn't seem to give you one.

Reply:
Wait, is this item intended for the synths? But they don't seem to spawn with it (or an autosurgeon)...

Reply:
this is intended for the synths so they can have some unique augments handed out by the leader. the leader spawns with the autodoc!